components/sections/StarOnGithub.tsx

```diff
@@ -0,0 +1,27 @@
+import { useLocalStorage } from 'react-use'
+
+const StarOnGithub = () => {
+	const [starOpen, setStarOpen] = useLocalStorage<boolean>('star-open', true)
```

Comment:
This doesn't work for me.

If the value is not already set in local storage, this indeed defaults it to `true`. Then on clicking the close button, it updates the value to false and triggers a re-render of this component, setting its display to `none`. However, if the value `false` is already in storage, the component still draws and clicking the button does not close it.

I suspect this is related to [this][1]. SSR initially renders the component with the stated default in `useLocalStorage` then hydration updates the state without going through `setStarOpen`, thus ignoring a rerender.

It seems the react-use project is not maintained anymore. Can we confirm where the issue with `useLocalStorage` comes from? And if it comes from react-use playing badly with Next, can we switch over to something actively maintained?

[1]: https://github.com/streamich/react-use/issues/2052